src/svlangserver.ts

```diff
@@ -113,6 +113,9 @@ connection.onInitialize((params: InitializeParams) => {
         else if (clientName.startsWith("emacs")) {
             svindexer.setClientDir(".emacs");
         }
+        else if (clientName.startsWith("Neovim")) {
+            svindexer.setClientDir(".nvim");
```

Comment:
Hey again @henry-hsieh :wave: (I found this while reviewing the server for addition in https://github.com/williamboman/nvim-lsp-installer/pull/762). Correct me if I'm wrong, but having a `.nvim/` directory is not an established convention for Neovim, I'm not sure it's something that should be assumed. I'm not sure what a "svindexer's client dir" is, but maybe it'd be better to use the default `.svlangserver`?

Reply:
Hi @williamboman,
You're right. Use default `.svlangserver` do the same tricks, too. It's used to let svlangserver detect the project root, and start to index the symbols.
I could open a new PR for this and possible modification of neovim/nvim-lspconfig#1948 later.